src/components/DashboardHome/__snapshots__/Sidebar.test.jsx.snap

```diff
@@ -14,27 +14,51 @@ exports[`<Sidebar /> renders correctly 1`] = `
       className="list-unstyled"
     >
       <li>
-        <span
-          aria-hidden={true}
-          className="fa fa-file mr-2 text-primary"
-          id="Icon1"
-        />
+        <svg
+          aria-hidden="true"
+          className="svg-inline--fa fa-file fa-w-12 mr-2 text-primary"
+          data-icon="file"
+          data-prefix="fas"
+          focusable="false"
+          role="img"
+          style={Object {}}
+          viewBox="0 0 384 512"
+          xmlns="http://www.w3.org/2000/svg"
```

Comment:
Just to be curious, can we use some built-in component from some standard library like https://www.npmjs.com/package/@fortawesome/fontawesome-svg-core ... Just dropping the comment here, cuz I am not comfortable with `xmlns="http://www.w3.org/2000/svg"` and path things :P

Reply:
@mushtaqak This SVG is being generated from `FontAwesomeIcon` provided by `@fortawesome/react-fontawesome`, which uses `@fortawesome/fontawesome-svg-core`.